### ISSUE TITLE
Not all projects are available in Project dropdown when using multiple workspaces

### ISSUE THREAD

**Issue by WimObiwan** [toggl/toggldesktop#1555]
I'm pretty sure that it only happens when:
* The user is member of multiple worskspaces, AND
* Some project names are the same in both projects

This problem doesn't exist on the Web GUI.

**Comment by amulware** [comment]
Can you specify which operating system this is about? Or does it happen on multiple/all (assuming you can/have tried that).

**Comment by WimObiwan** [comment]
The only OS I tried with, is **Windows 8.1 Pro**.

**Comment by amulware** [comment]
I can reproduce some issues, and I am fairly confident that this is something in the library. Will do some more testing to confirm.

**Comment by amulware** [comment]
This is what I have been able to reproduce:

I have two workspaces that share some project and client names, and clients with the same name have both differently and equal project names.

As reported above, if there are projects with the same name (AND clients of the same name/no client), only one of these shows up. It seems to be arbitrary which shows up and which does not (seen different combinations just by adding a new project and reloading the app).

I did some debugging and can also confirm that the problem does not occur within the UI. The auto completion list handed over by the library is already missing the entries.
